TUnit.Assertions/Attributes/AssertionType.cs

```diff
@@ -0,0 +1,14 @@
+﻿// ---------------------------------------------------------------------------------------------------------------------
+// Imports
+// ---------------------------------------------------------------------------------------------------------------------
+
+namespace TUnit.Assertions;
+
+// ---------------------------------------------------------------------------------------------------------------------
+// Code
+// ---------------------------------------------------------------------------------------------------------------------
+public enum AssertionType {
+    Undefined,
```

Comment:
What would undefined be?

And going forwards we could I guess expand this with:

`Does`, `DoesNot`, `Has`, etc ?